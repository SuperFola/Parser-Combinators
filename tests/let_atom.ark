(let aaaaaaa 1)
# test comment
#comment###
(
let
b
"12"
)