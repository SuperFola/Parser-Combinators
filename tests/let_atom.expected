( Keyword:let Symbol:aaaaaaa Number:12 )
( Keyword:mut Symbol:b Number:13 )
( Keyword:set Symbol:c String: )
( Keyword:let Symbol:b String:12 )